### PR TITLE
interactive bots: Run the message_info bot for ddxtanx.

### PR DESCRIPTION
**Pull request by ddxtanx**
GCI Link - https://codein.withgoogle.com/dashboard/task-instances/5275590326222848/

### CODE REVIEW

File: interactive-bots/message_info/ddxtanx/notes.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+During the development process there was one minor hitch I got into,
```
Comment (tommyip) [inline interactive-bots/message_info/ddxtanx/notes.md]: Can you line wrap all text files to 79 characters?

**Comment by tommyip** [comment]
@ddxtanx awesome! Can you now squash the two commits to one so I am merge and approve your task? Ask on https://chat.zulip.org/#narrow/stream/GCI.20help/ if you are unsure on how to do this. (Also keep your commit title short so that it doesn't overflow to the body).

**Comment by ddxtanx** [comment]
Commits should be squashed!

**Comment by tommyip** [comment]
Awesome! Thanks @ddxtanx .

**Pull request merged**